### DISCUSSION TITLE
API Calls and Version Controlls

### DISCUSSION THREAD

**Discussion by ankit-airmeet** [WeblateOrg/weblate#6137] [General]
Have multiple set of questions:-
1. Do you expose all APIs where we can easily apply CRUD operations on Key Values?
2. Do you expose API where we can Migrate Versions from one to Another Automatically?
3. Do you necessarily need Git Repo Access to Automate the Localization Process for an Application or it can be done with custom API Calls?
4. You any CLI exposed to do some Custom Operations?
5. How is your Downtime for general Maintenance?

**Comment by nijel** [comment]
1-2. The API is documented here: https://docs.weblate.org/en/latest/api.html
3. Remote Git repository is not necessary, see https://docs.weblate.org/en/latest/vcs.html#local-files
4. Server side command line is documented at https://docs.weblate.org/en/latest/admin/management.html
5. Most of the maintenance happens without noticeable service interruption, see https://status.weblate.org/ for our past outages.

Should you have any specific pre-sale question, don't hesitate to ask us on sales@weblate.org.
